mllaunchpad/cli.py

```diff
@@ -15,104 +17,117 @@
 from mllaunchpad.api import generate_raml
 from mllaunchpad.api import ModelApi
 
-HELP_STRING = """
-Parameters:
--h          / --help                  : Print this help
--v          / --version               : Print the version
--t          / --train                 : Run training, store model and metrics
--r          / --retest                : Retest model, update metrics
--a          / --api                   : Run API Server (Debug Mode!)
--c <file>   / --config=<file>         : Config file to use
--l <file>   / --logconfig=<file>      : Log config file to use
--g <datsrc> / --generateraml=<datsrc> : Generate and print RAML template
-                                        from data source name
-"""
```

Comment:
I don't see anymore the aliases like '-t' for '--train'. Does it mean that we always have to use full 'train'? It doesn't matter for the review but just for my clarity ;)

Reply:
Good point!  I‘ll push a commit where the commands (train, predict, api, generate-raml) are matched from the beginning (as long as they are unique), so you can use e.g. t, tr, ... which mat vg es train uniquely, g, gen, ... which matches generate-raml uniquely, a for api and so on.